Mathlib/LinearAlgebra/Matrix/Spectrum.lean

```diff
@@ -55,6 +56,33 @@ lemma mulVec_eigenvectorBasis (j : n) :
       congr(⇑$((isHermitian_iff_isSymmetric.1 hA).apply_eigenvectorBasis
         finrank_euclideanSpace ((Fintype.equivOfCardEq (Fintype.card_fin _)).symm j)))
 
+/--Eigenvalues of a Hermitian Matrix, coerced, belong to the spectrum of the assoc.toEuclideanLin -/
+theorem eigenvalue_mem_toEuclideanLin_spectrum_RCLike (i : n) :
+    (RCLike.ofReal ∘ hA.eigenvalues) i ∈ spectrum 𝕜 (toEuclideanLin A) :=
```

Comment:
I think this should be written as
```suggestion
theorem ofReal_eigenvalue_mem_spectrum_toEuclideanLin (i : n) :
    hA.eigenvalues i ∈ spectrum 𝕜 (toEuclideanLin A) :=
```
where hopefully the coercion is inserted automatically

Reply:
Implemented name change, but unfortunately if I remove the explicit coercion, the result won't compile.

Reply:
I pushed a fix